package.json

```diff
@@ -110,6 +116,9 @@
     "@vuepress/plugin-back-to-top": "1.4.1",
     "codecov": "^3.7.1",
     "cz-conventional-changelog": "3.1.0",
+    "death": "^1.1.0",
+    "docker-compose": "^0.23.6",
+    "dotenv-cli": "^4.0.0",
```

Comment:
Are these packages used?

I think the `docker-compose` you run ex during `yarn dev` is the `docker-compose` being installed locally on the host machine. I might be wrong though.

Reply:
The `death` and `docker-compose` ones aren't used - they were leftovers from previous experimentation. Nice catch!